packages/db/src/core/integration/index.ts

```diff
@@ -113,6 +106,36 @@ function astroDBIntegration(): AstroIntegration {
 						server.restart();
 					}
 				});
+				// Wait for dev server log before showing "connected".
+				await new Promise((resolve) => setTimeout(resolve, 100));
+				logger.info(
+					connectToStudio ? 'Connected to remote database.' : 'New local database created.'
+				);
+				const seedFileUrls = SEED_DEV_FILE_NAME.map(
```

Comment:
Does this also need to account for integration seed files? Right now it looks like it’s only our static array of project ones?

Reply:
Ah, that's a good point! This is only for _eagerly_ loading when a seed file changes, so it shouldn't break existing behavior. Happy to add if it's easy enough

Reply:
Update: this will work for relative paths but _not_ package ids. Since eagerly reloading is a nice-to-have, I think this is a fine compromise